GUI/GUI.py

```diff
@@ -181,7 +181,11 @@ def add_spacings(n: int = 2) -> None:
             imgui.spacing()
 
     @staticmethod
-    def add_tooltip(text: str) -> None:
+    def add_tooltip(text: str, width: int = 300) -> None:
         """Add a tooltip to the previous element."""
-        if imgui.is_item_hovered():
-            imgui.set_tooltip(text)
+        if imgui.is_item_hovered(flags=imgui.HoveredFlags_.delay_normal) and imgui.begin_tooltip():
```

Comment:
What does this change do specifically? A different way to do tooltips?

Reply:
Yes, instead of the one-liner it uses the full begin and end tooltip which allows for adding a delay, word wrapping and control over the width of tooltips. 
Now they have a default width of 300 (pixels i guess) but that can be overwritten by setting the (named) width argument. See the checkpoint list for an example on that.